ts/components/ui/SectionList.tsx

```diff
@@ -1,9 +1,15 @@
 import { connectStyle } from "native-base-shoutem-theme";
 import mapPropsToStyleNames from "native-base/src/utils/mapPropsToStyleNames";
+import * as React from "react";
 import { SectionList as RNSectionList } from "react-native";
+import { ComponentProps } from "../../types/react";
+
+interface Props extends ComponentProps<RNSectionList<any>> {
+  padded: boolean;
+}
 
 export const SectionList = connectStyle(
   "UIComponent.SectionList",
   {},
   mapPropsToStyleNames
-)(RNSectionList);
+)<typeof RNSectionList, React.ComponentClass<Props>>(RNSectionList);
```

Comment:
I'm not sure about this, isn't the theming supposed to work only on Native Base components? - cc @fpersico